packages/bulk-edit/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "@theia/bulk-edit",
+  "version": "1.6.0",
+  "description": "Theia - Bulk Edit Extension",
+  "dependencies": {
+    "@theia/core": "^1.6.0",
+    "@theia/editor": "^1.6.0",
+    "@theia/monaco": "^1.6.0",
+    "@types/p-queue": "^2.3.1",
+    "@theia/filesystem": "^1.6.0",
+    "@theia/navigator": "^1.6.0",
+    "@theia/workspace": "^1.6.0",
```

Comment:
Please sort these dependencies (it is done automatically during releases but it will result in changes).

Reply:
Fixed